Spring, first try

HKV203 opened the pull request:
It uses Spring Boot and shows the book list in the "Change Book Availability" part. 

It uses my own jpa starter now.

Still I don't really get what you meant about the second starter. I use Postgresql.

Commit:
Spring, first try

Alexmansar commented on pom.xml:
```diff
@@ -16,6 +16,23 @@
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>4.0.1</version>
+        </dependency>
```
do we need it? we use boot,  so can use srarters

Alexmansar commented on pom.xml:
```diff
@@ -91,6 +104,31 @@
+            <version>3.4.0</version>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/org.apache.tomcat.embed/tomcat-embed-jasper -->
+        <dependency>
+            <groupId>org.apache.tomcat</groupId>
+            <artifactId>tomcat-jasper</artifactId>
+            <version>11.0.1</version>
+        </dependency>
```
also in our starters

HKV203 replied:
@Alexmansar it isn't. Maybe you meant spring-boot-starter-tomcat that we have in spring-boot-starter-web? https://stackoverflow.com/questions/42154614/springboot-embedded-tomcat-and-tomcat-embed-jasper

Alexmansar replied:
I general we don't need extra dependancy here) your sring boot already has it in spring-boot-starter-web
so just use spring-boot-starter-web to build your app)

Alexmansar replied:
sp ypu ca just remove old solution with servlets ans tomcat, and use spring))

HKV203 replied:
@Alexmansar tomcat-jasper helps me to connect my Controller class and jsp files. If I take out this dependency, I get the error page instead of index.jsp and all the other pages my browser downloads  instead of opening

HKV203 replied:
Even though I have spring-boot-starter-web in my dependencies

Alexmansar replied:
Ok) you may be right)) jsp is not popular these days, and I've worked a long time ago)))

Alexmansar commented on src/main/java/com/andersen/bookstore/Main.java:
```diff
@@ -1,11 +1,21 @@
+    @Override
+    protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
+        return builder.sources(Main.class);
+    }
```
do we need it?

Alexmansar commented on src/main/java/com/andersen/bookstore/Main.java:
```diff
@@ -1,11 +1,21 @@
-    public static void main(String[] args) {
-        new Controller().runConsole();
+@SpringBootApplication
+public class Main extends SpringBootServletInitializer {
```
extends SpringBootServletInitializer 
and this?

Alexmansar commented on src/main/java/com/andersen/bookstore/controller/BookController.java:
```diff
@@ -0,0 +1,54 @@
+            } else if ("no".equals(isAvailable)) {
+                bookService.updateBookAvailability(number, false);
+            }
+        }
```
this is service work

Alexmansar reviewed (changes requested):
@Mopgonpblr Hi) please, fix some comments

Commit:
1. servlets and servlet-api in pom.xml are deleted
2. Main class simplified
3. Book Availability logic is transferred from Controller class to Service class
4. New starter that provides order list from JSON file
